tests/ui/unsafe-binders/simple.rs

```diff
@@ -1,7 +1,8 @@
+//@ check-pass
+
 #![feature(unsafe_binders)]
 //~^ WARN the feature `unsafe_binders` is incomplete
 
 fn main() {
     let x: unsafe<'a> &'a ();
```

Comment:
Why does this compile without an assignment?

Reply:
Becuase variables don't need to be assigned